src/libraries/Microsoft.PowerFx.Core/Texl/Builtins/Language.cs

```diff
@@ -0,0 +1,32 @@
+﻿// Copyright (c) Microsoft Corporation.
+// Licensed under the MIT license.
+
+using System.Collections.Generic;
+using System.Collections.ObjectModel;
+using System.Linq;
+using Microsoft.PowerFx.Core.Functions;
+using Microsoft.PowerFx.Core.Localization;
+using Microsoft.PowerFx.Core.Types;
+using Microsoft.PowerFx.Core.Utils;
+
+namespace Microsoft.PowerFx.Core.Texl.Builtins
+{
+    internal class LanguageFunction : BuiltinFunction
+    {
+        public override bool IsSelfContained => true;
+
+        public override bool HasPreciseErrors => true;
+
+        public override bool SupportsParamCoercion => true;
```

Comment:
>public override bool SupportsParamCoercion => true; [](http://example.com/codeflow?start=8&length=51)

Please remove it - it doesn't have any parameters

Reply:
same as above

Reply:
removed.